R/uvhydrograph-data.R

```diff
@@ -409,8 +414,10 @@ hasUpchainSeries <- function(reportObject) {
 #' @param isDischarge whether or not the primary series is discharge
 #' @return the lims object that to be applied to the primary plot
 calculatePrimaryLims <- function(primarySeriesList, isDischarge) {
-  if(!is.null(primarySeriesList[['corrected']])){
+  if(!isEmptyOrBlank(primarySeriesList[['corrected']][['points']][['value']])){
     lims <- calculateLims(primarySeriesList[['corrected']][['points']])
+  } else if(primarySeriesList[['useEstimated']]){
+    lims <- calculateLims(primarySeriesList[['estimated']][['points']])
   } else {
     lims <- calculateLims(primarySeriesList[['uncorrected']][['points']])
   }
```

Comment:
I guess it should just be noted in that new issue that this may or may not be necessary to keep.